cmd/autoheal/alerts_worker.go

```diff
@@ -192,6 +193,13 @@ func (h *Healer) runRule(rule *autoheal.HealingRule, alert *alertmanager.Alert)
 		return nil
 	}
 
+	// Increment the metric of requested heales.
+	h.incrementRequestedActions(
+		reflect.TypeOf(action).Name(),
```

Comment:
The above currently returns an empty string.

Reply:
That is because in this context the type of `action` is a pointer, and the name of pointer types happens to be empty. Try this:

```go
reflect.TypeOf(action).Elem().Name()
```

Reply:
Works, thanks!